src/notification-marker/notification-marker.scss

```diff
@@ -56,3 +50,14 @@
     }
   }
 }
+
+@mixin iui-notification-marker-hover {
```

Comment:
Why mixin instead of having stylings inside of the main mixin?

Reply:
The main mixin is only used by `/notification-marker/classes.scss` but `@mixin iui-notification-marker-hover` is used within `/button/default.scss`.  Button targets `&:enabled:hover, &:enabled:active, &:focus-visible`, but if we wanted to use the notification marker within a different component, that component may not target `:enabled:hover` for example.